controllers/process.js

```diff
@@ -50,4 +47,57 @@ module.exports = ((req, res) => {
       ua.event('Entries', 'New entry error').send()
     }
   })
-})
+}
+
+function verifyCAPTCHA(req, res) {
+  return new Promise((resolve, reject) => {
+    staticman
+      .getSiteConfig()
+      .then(siteConfig => {
+        if (!siteConfig.get('reCaptcha.enabled')) {
+          return resolve()
+        }
+
+        if(
+          req.body.options.reCAPTCHA === undefined ||
+          req.body.options.reCAPTCHA.siteKey === undefined ||
+          req.body.options.reCAPTCHA.encryptedSecret === undefined
+        ) {
+          return reject('Missing reCAPTCHA API credential.')
+        }
+
+        recaptcha.init(
```

Comment:
Sorry for being picky, but I'm not a fan of having function calls across multiple lines. If it gets too long for a single line, could you create variables with the arguments and use them in the function call instead?